sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcSourceSuite.scala

```diff
@@ -35,6 +35,8 @@ import org.apache.spark.SPARK_VERSION_SHORT
 import org.apache.spark.sql.{Row, SPARK_VERSION_METADATA_KEY}
 import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.test.SharedSQLContext
+import org.apache.spark.sql.types.{LongType, StringType, StructType}
+import org.apache.spark.sql.types.DataTypes._
```

Comment:
Could you remove these changes because we removes the test cases?

Reply:
Good point, removed. I'm surprised the linter/compiler didn't complain about these unused imports, it seems pretty strict otherwise.